public/docs/dart/latest/guide/server-communication.jade

```diff
@@ -1,11 +1,529 @@
 include ../_util-fns
+:marked
+  [HTTP](https://tools.ietf.org/html/rfc2616) is the primary protocol for browser/server communication.
+.l-sub-section
+  :marked
+    The [`WebSocket`](https://tools.ietf.org/html/rfc6455) protocol is another important communication technology;
+    we won't cover it in this chapter.
+:marked
+  Modern browsers support two HTTP-based APIs: 
+  [XMLHttpRequest (XHR)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and 
+  [JSONP](https://en.wikipedia.org/wiki/JSONP). A few browsers also support
+  [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). 
+  
+  The Angular HTTP client library simplifies application programming of the **XHR** and **JSONP** APIs
+  as we'll learn in this chapter covering:
+  
+  [Http client sample overview](#http-client)<br>
+  [Fetch data with http.get](#fetch-data)<br>
+  [RxJS Observable of HTTP Responses](#rxjs)<br>
```

Comment:
No, sorry. I will remove this line. I don't know also if we are going to be able to use `RxDart` either.